scripts/pretty-printers/gdb/install.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+# If you need to apply this manually, copy/paste the below code into a .gdbinit
+# of your choice and replace {0} with the path to this folder.
+file_contents = """
+python
+import sys
+import os
+
+pretty_printer_folder = '{0}'
+if os.path.exists(pretty_printer_folder):
+    sys.path.insert(1, pretty_printer_folder)
+    import auto_load
+    auto_load.load_pretty_printers()
+end
```

Comment:
:question: What is with this `end`? If pasted into an `__init__.py` then this end breaks things, is it ending the python section?

Reply:
Yeah, specific syntax for the .gdbinit file to state the end of a python block (where 'python' states the beginning).